scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -4200,23 +4200,22 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
           (anonymousName, List.empty, List.empty)
       }
 
-    val decltpe = startModType()
+    val decltype = startModType()
```

Comment:
still renamed... i know tpe is weird but that's what it's called in 100 other places in scalameta